portal/server/routes/contract/about/supplierPage.js

```diff
@@ -128,7 +137,7 @@ const supplierSubmissionDetailsFields = [
 router.post('/contract/:_id/about/supplier', provide([INDUSTRY_SECTORS]), async (req, res) => {
   const { industrySectors } = req.apiData;
   const { _id, userToken } = requestParams(req);
-  let submissionDetails = constructPayload(req.body, supplierSubmissionDetailsFields);
+  let submissionDetails = constructPayload(req.body, supplierSubmissionDetailsFields,true);
```

Comment:
same q about allowing empty properties here

Reply:
It will insert the empty properties but when we insert the value, the property will update and code will check the validation also for all the pages.